rag/flow/extractor/extractor.py

```
@@ -74,6 +119,78 @@ async def _build_TOC(self, docs):
+            # not-fitted messages anyway (validate_fitted_messages() already
+            # means the request is degenerate, e.g. an empty user turn).
+            logging.warning("Extractor document_level: %s; skipping field %r.", fit_error, self._param.field_name)
+            return
+
+        parsed = None
+        for _ in range(self._param.max_retries + 1):
+            result = await self.chat_mdl.async_chat(msg_fit[0]["content"], msg_fit[1:], self._param.gen_conf())
+            try:
+                candidate = json_repair.loads(result)
+            except Exception:
+                continue
```

Ruff (0.15.21)

[error] `try`-`except`-`continue` detected, consider logging the exception

(S112)